Pkgfile

```diff
@@ -103,9 +103,9 @@ vars:
   flex_sha512: e9785f3d620a204b7d20222888917dc065c2036cae28667065bf7862dfa1b25235095a12fd04efdbd09bfd17d3452e6b9ef953a8c1137862ff671c97132a082e
 
   # renovate: datasource=git-tags extractVersion=^gawk-(?<version>.*)$ depName=git://git.savannah.gnu.org/gawk.git
-  gawk_version: 5.1.1
-  gawk_sha256: d87629386e894bbea11a5e00515fc909dc9b7249529dad9e6a3a2c77085f7ea2
-  gawk_sha512: 794538fff03fdb9a8527a6898b26383d01988e8f8456f8d48131676387669a8bb3e706fa1a17f6b6316ddba0ebe653c24ad5dd769f357de509d6ec25f3ff1a43
+  gawk_version: 5.2.1
```

Comment:
this is a fixed version after broken 5.2.0

Reply:
finally :sweat_smile: